### ISSUE TITLE
Color problems with the newsletter sign-up pattern

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyfive#526]
**Description**

The color contrast ratio in the "Newsletter sign-up" pattern is not high enough in all combined style variations.
The text color on this cover block is set to `#000` which does not work for all accent-color 1 background colors in all palettes.

**I am wondering, if one solution would be to change the order of the palette so that accent-color 1 is always a light color...?**

**Step-by-step reproduction instructions**
Insert the "Newsletter sign-up" pattern.
Test the pattern with all combined style variations.

**Expected behavior**
For the text in the pattern to be readable.
WCAG 2.0 level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text.

**Note**
Please remember that automated tools are not able to get the correct colors from the cover block, some manual testing is required.

Midnight:
The background and the text color of the heading and text has a contrast ratio of 2.94:1.

![Image](https://github.com/user-attachments/assets/5a05f679-5a88-4f0b-b630-7e5347918003)

Evening:
The background and the text color of the heading and text has a contrast ratio of 3.99:1.
This means that the heading passes, since it has a larger font size, but the paragraph does not pass.

![Image](https://github.com/user-attachments/assets/4091e3b1-92e7-4925-82c4-36321874635e)

**Comment by carolinan** [comment]
> I am wondering, if one solution would be to change the order of the palette so that accent-color 1 is always a light color...?

Only the default palette was sorted from light to dark as part of https://github.com/WordPress/twentytwentyfive/pull/440 because at that time, it proved quiet complex.

**Comment by carolinan** [comment]
Here are partial screenshots of the Evening variation and two other patterns that use accent-color 1. The text is difficult to read.

![Image](https://github.com/user-attachments/assets/ceefa620-3a4f-433e-a7cb-6d8bac0f0908)
![Image](https://github.com/user-attachments/assets/2ad6839a-9521-47c7-aed0-c80c9b579255)

I did not find **one** accent color that will work with **both** these patterns.

**Comment by carolinan** [comment]
Another important thing to consider is that I believe the color is meant to be the same as in the footer?

This is the home template for the "news blog with featured posts grid"
![Image](https://github.com/user-attachments/assets/e85aea17-ffd2-45b1-bdd4-7d41cdc25ce4)
